etc/ci/update_brew.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cd "$(dirname $0)/../.."
+
+PACKAGEPATH=`ls -t target/servo-????-??-??.tar.gz | head -n 1`
+PACKAGENAME=`basename $PACKAGEPATH`
+VERSION=`echo $PACKAGENAME| sed -n "s/servo-.*\([0-9]\{4\}\)-\([0-9]\{2\}\)-\([0-9]\{2\}\).tar.gz*/\1.\2.\3/p"`
+PACKAGEURL="http://people.mozilla.com/~prouget/graphene/brew/$PACKAGENAME" 
+SHA=`shasum -a 256 $PACKAGEPATH | sed -e 's/ .*//'`
+
+if [[ -z $VERSION ]]; then
+  echo "Package doesn't havent the right format: $PACKAGENAME"
+  exit 1
+fi
+
+scp $PACKAGEPATH prouget@people.mozilla.com:public_html/graphene/brew/
+
+echo "Package successfuly uploaded to $PACKAGEURL"
+
+tmp_dir=`mktemp -d -t homebrew-servo`
+cd $tmp_dir
+echo $tmp_dir
+
+echo "Cloning"
+git clone https://github.com/paulrouget/homebrew-servo
```

Comment:
To clarify, considering `homebrew` uses git repos as package repos, I would include the `servo-bin.rb.in` file in the `servo` repo, and push the rendered nightly versions to another repo.